kubernetes/annotations_and_labels.md

```diff
@@ -41,18 +41,30 @@ This page defines common annotations and labels we set in Kubernetes objects.
   managed by `chart-operator`.
 - `release.giantswarm.io/version` - value should be Giant Swarm release
   version, e.g. `release.giantswarm.io/version=2.3.0`.
-- `OPERATOR.giantswarm.io/version` - value should be an operator version, e.g.
-  `kvm-operator.giantswarm.io/version=1.0.0`. This is not the same as release
-  version. This value can be the same in multiple releases. This label is used
-  by the operator to recognize which object it should reconcile. When set on
-  nodes it is used to set that information in the status with the
-  statusresource.
+- `app.giantswarm.io/version` - value should be an app/operator version as
+  defined in `project.go`, e.g. `app.giantswarm.io/version=1.0.0`. This is not
+  the same as release version. This value can be the same in multiple releases.
+  This label is used by the operator to recognize which object it should
+  reconcile. When set on nodes it is used to set that information in the status
+  with the statusresource.
```

Comment:
Maybe we should also add a word that this should match `appVersion` in Chart.yaml.

Reply:
Done.